conftest.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--use-requests-cache",
+        action="store_true",
+        help="Use requests_cache for caching web API calls"
+    )
+
+
+import pytest
+@pytest.fixture(scope='session', autouse=True)
+def use_requests_cache(request):
+    """Use requests_cache for caching web API calls. 
+       Enabled with --use-requests-cache flag:
+
+          pytest --use-requests-cache
+
+       Notes:
+       - The cache persists for 12 hours.
+       - Cache does not working for invocations of manubot commands as subprocess.
+    """
+    if request.config.getoption("--use-requests-cache"): 
+        start_caching(hours=12)
+
+
+def start_caching(**kwargs):
+    from datetime import timedelta
+    from requests_cache import install_cache
+    if not kwargs:
+        kwargs = dict(hours=1)
```

Comment:
What if we just put `hours=1` or something similar in the function definition? This seems like extra logic that never ends up being used, given the rest of the implementation.

Unless is the idea that code elsewhere may call `start_caching`?

Reply:
Still not sure why we can't just make `hours` an argument and remove the unused `**kwargs` logic and processing.